lib/action.ts

```diff
@@ -62,7 +62,7 @@ export const createAction = async (
   });
 };
 
-export const fileFeedEventFromAction = (action: Action): FileFeedEvent => {
+export const fileFeedEventFromAction = (action: any): FileFeedEvent => {
```

Comment:
No any's, give it a new type that makes sense

Reply:
type added